test/sagemaker_tests/pytorch/inference/requirements.txt

```diff
@@ -12,7 +12,8 @@ PyYAML==5.4
 sagemaker>=2,<3
 six==1.12.0
 requests==2.24.0
-torchvision
+torch==1.11
+torchvision==0.12
```

Comment:
Does pinning here mean that we'll have to periodically update this version?

Reply:
I believe so, yes. Unfortunately, I wasn't able to deep dive and figure out why the torch import was failing when torch isn't pinned here. It should have been installed as a dependency of torchvision at any compatible version, but apparently that doesn't work as expected.

Reply:
I'm confused on this pinning here - why torch 1.11, and what happens if we try to run this on 1.12 or 1.13?

In what circumstances do these get updated?

Reply:
@arjkesh I just upgraded it to the earliest supported version as of now. This would need to be updated when tests fail due to being unable to generate the saved model to be used for SM PT inference tests.

Reply:
Unpinned and it works. I guess installing torch as a dependency of torchvision was the problem.